### PR TITLE
SSCS-3829 Removed surname validating regex

### PR DESCRIPTION
**Pull request by guioconnor**
This regex checks against patterns for surnames, but given the input is not used anywhere where code injection would be possible, it's better to remove the pattern check altogether and allow the users to use any character

The regex validation has neither a security nor a usability purpose. A user is likely to input their name as accurately as possible and when they don't, the consequences are minimal and very low severity. 

The experience of being told a name is invalid is detrimental to the user experience, it's more useful simply being told the surname doesn't match the records.

screen shot 2018-09-11 at 21 10 47

Make sure the branch has the JIRA number in it (eg. sscs-1111-do-amazing-things)

- Add a full description of what this change does and, most importantly, why.
- Include links to documentation, blog posts or even stack overflow questions that helped you.
- If it adds dependencies mention them.
- If it changes the UI consider including before and after screenshots.

## Review checklist
- [ ] Has a test analyst reviewed this change?
- [x] Has another developer reviewed this change?
- [ ] Has a designer review this change?
- [ ] Does it have good test coverage and have you run the tests?
- [ ] Have you run the app and verified it works?
- [x] Does the commit log tell a clear concise story? (If it's one logical change it should be one commit)
- [ ] Does it follow our ubiquitous language?
- [ ] Does it follow our [good app guidelines](http://git.reform/sscs/readme/blob/master/good-app-guidelines.md)?

### CODE REVIEW

File: test/unit/middleware/validateSurname.test.js [hunk 1]
```diff
@@ -53,14 +53,5 @@ describe('validateSurname.js', () => {
...
-
-    it('calls res.render when the field is invalid', () => {
-      fields.surname.value = '12345';
-      fields.surname.errorHeading = validateSurnameContent.surname.errors.notValidHeading;
-      fields.surname.errorMessage = validateSurnameContent.surname.errors.notValidHeading;
-      req.body.surname = '12345';
-      validateSurname(req, res, next);
-      expect(res.render).to.have.been.calledWith('validate-surname', { id, fields, originalPage });
-    });
   });
```
Comment (iguannago) [inline test/unit/middleware/validateSurname.test.js]: It would be good to have a test confirming you can actually enter any character in the surname. And another test that checks that the surname has to be minimum 2 chars, right?

**Comment by guioconnor** [comment]
Indeed, added now.

**Review by barbaracassani — APPROVED** [review]

**Review by Edlyne — APPROVED** [review]
QA APPROVED.
